streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -77,79 +81,27 @@
-        this.getTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "get",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        this.deleteTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "delete",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        this.putAllTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "put-all",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        this.allTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "all",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        this.rangeTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "range",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        this.flushTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "flush",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
-        final Sensor restoreTime = this.metrics.addLatencyAndThroughputSensor(
-            taskName,
-            metricScope,
-            name,
-            "restore",
-            Sensor.RecordingLevel.DEBUG,
-            tagKey, taskName);
+        putTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "put", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        putIfAbsentTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "put-if-absent", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        putAllTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "put-all", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        getTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "get", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        allTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "all", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        rangeTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "range", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        flushTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "flush", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        deleteTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "delete", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
+        final Sensor restoreTime = createTaskAndStoreLatencyAndThroughputSensors(DEBUG, "restore", metrics, metricsGroup, taskName, name(), taskTags, storeTags);
```

Comment:
These lines are a little long, but they're also very uniform. Should I split them?

Reply:
I am fine both ways.